src/jdk.crypto.cryptoki/share/classes/sun/security/pkcs11/Config.java

```diff
@@ -500,9 +513,13 @@ private void parse() throws IOException {
             case "nssOptimizeSpace"->
                 nssOptimizeSpace = parseBooleanEntry(st.sval);
             default->
-                throw new ConfigurationException
-                        ("Unknown keyword '" + st.sval + "', line " +
-                        st.lineno());
+                if ("tokenLabel".equalsIgnoreCase(st.sval)) {
+                    tokenLabel = parseStringEntry("tokenLabel");
+                } else {
+                    throw new ConfigurationException
+                            ("Unknown keyword '" + st.sval + "', line " +
+                                    st.lineno());
+                }
```

Comment:
The braces I suggested around this code are required; it doesn't compile without them.